mate_menu/plugins/applications.py

```diff
@@ -285,7 +307,8 @@ def __init__(self, mateMenuWin, toggleButton):
 
         for mainitems in [ "mate-applications.menu", "mate-settings.menu" ]:
             mymenu = Menu( mainitems )
-            mymenu.tree.add_monitor( self.menuChanged, None )
+            #mymenu.tree.add_monitor( self.menuChanged, None )
```

Comment:
The old line can be dropped, no need to comment the code that won't work with new API...